{
  "comments": [
    {
      "key": {
        "uuid": "a2f8c0e6_fbadc11e",
        "filename": "ci-scripts/infra-setup/roles/rrcockpit/files/telegraf/requirements.txt",
        "patchSetId": 5
      },
      "lineNbr": 2,
      "author": {
        "id": 673
      },
      "writtenOn": "2020-12-16T11:19:36Z",
      "side": 1,
      "message": "I think we should update this dependency too.",
      "range": {
        "startLine": 2,
        "startChar": 0,
        "endLine": 2,
        "endChar": 16
      },
      "revId": "990c0889dd19d4103fa4892580a434dd2cb81499",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "19ff8d89_7a252565",
        "filename": "ci-scripts/infra-setup/roles/rrcockpit/files/telegraf/requirements.txt",
        "patchSetId": 5
      },
      "lineNbr": 2,
      "author": {
        "id": 192
      },
      "writtenOn": "2020-12-16T11:25:19Z",
      "side": 1,
      "message": "yeah i thought the same but i am doing them one at a time to see what is actually necessary\n\nthe last thing it complained about was requests, so it seems it was able to fulfill diskcache 3.0.6?\n\nwaiting to see if pyyaml breaks i expect it might",
      "parentUuid": "a2f8c0e6_fbadc11e",
      "range": {
        "startLine": 2,
        "startChar": 0,
        "endLine": 2,
        "endChar": 16
      },
      "revId": "990c0889dd19d4103fa4892580a434dd2cb81499",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "54743a40_7064c55f",
        "filename": "ci-scripts/infra-setup/roles/rrcockpit/files/telegraf/requirements.txt",
        "patchSetId": 5
      },
      "lineNbr": 2,
      "author": {
        "id": 192
      },
      "writtenOn": "2020-12-16T11:25:45Z",
      "side": 1,
      "message": "there for requests fyi         * https://logserver.rdoproject.org/58/31358/4/check/tox-py36-ci-config/7652cef/job-output.txt\n        * 2020-12-16 10:37:57.041054 | rdo-centos-8 | ERROR: Could not find a version that satisfies the requirement requests\u003d\u003d2.19.1",
      "parentUuid": "19ff8d89_7a252565",
      "range": {
        "startLine": 2,
        "startChar": 0,
        "endLine": 2,
        "endChar": 16
      },
      "revId": "990c0889dd19d4103fa4892580a434dd2cb81499",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "831226c1_17029b4e",
        "filename": "ci-scripts/infra-setup/roles/rrcockpit/files/telegraf/requirements.txt",
        "patchSetId": 5
      },
      "lineNbr": 6,
      "author": {
        "id": 673
      },
      "writtenOn": "2020-12-16T11:19:36Z",
      "side": 1,
      "message": "ditto.",
      "range": {
        "startLine": 6,
        "startChar": 0,
        "endLine": 6,
        "endChar": 11
      },
      "revId": "990c0889dd19d4103fa4892580a434dd2cb81499",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "d1949ac0_9d066b76",
        "filename": "ci-scripts/infra-setup/roles/rrcockpit/files/telegraf/requirements.txt",
        "patchSetId": 5
      },
      "lineNbr": 6,
      "author": {
        "id": 192
      },
      "writtenOn": "2020-12-16T11:37:13Z",
      "side": 1,
      "message": "yup will change this one now ERROR: No matching distribution found for pyyaml\u003d\u003d5.3\n        * https://logserver.rdoproject.org/58/31358/5/check/tox-py36-ci-config/33a0d3c/tox/py36-1.log",
      "parentUuid": "831226c1_17029b4e",
      "range": {
        "startLine": 6,
        "startChar": 0,
        "endLine": 6,
        "endChar": 11
      },
      "revId": "990c0889dd19d4103fa4892580a434dd2cb81499",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    }
  ]
}